### ISSUE TITLE
macos: unable to start mobile testing due to the inaccessible usbmuxd for the appium

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/appium#236]
if USBMUXD_SOCKET_ADDRESS not empty we have to join to it (macos<-linux)
```
socat UNIX-LISTEN:/var/run/usbmuxd,fork,reuseaddr,mode=777 TCP:$USBMUXD_SOCKET_ADDRESS &
```

if not - that's a linux and we have to strart our own:
```
socat TCP-LISTEN:22,reuseaddr,fork UNIX-CONNECT:/var/run/usbmuxd &
```

Note: how about making 2222 as default one for the 2nd use-case? i.e. for mcoud-device on Linux it should be appium:2222

**Comment by vdelendik** [comment]
done, changes are available in `zebrunner/appium:2.0.4`

**Comment by dhreben** [comment]
Verified with Mobile-iOS-Preferences and Safari tests
